content/en/docs/concepts/overview/working-with-objects/field-selectors.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Field Selectors
+weight: 60
+---
+
+_Field selectors_ enable you to [select Kubernetes resources](../kubernetes-objects) on the basis of the value of one or more resource fields. Here are some example field selector queries:
```

Comment:
> "../kubernetes-objects"

I'm not sue if we should avoid relative paths in docs. The fact is that this is rare. Having relative paths in docs is building implicit links between topics. When one topic is later moved to other places in the directory, these relative references are difficult to find and to fix.